dotnet/README.md

```diff
@@ -0,0 +1,24 @@
+# Hello, World in .NET
+
+This flake defines:
+
+* `nix run .#fantomas -- -r .` to run the [Fantomas](https://fsprojects.github.io/fantomas/) F# source formatter.
+* `nix develop . --command alejandra .` to run the [Alejandra](https://github.com/kamadorueda/alejandra) Nix source formatter.
+* `nix develop . --command markdown-link-check README.md` to check that this README's links are not broken.
+* `nix develop . --command bash -c "find . -type f -name '*.sh' | xargs shellcheck"` to check all shell scripts in this repository.
+* `nix run . --` to run the application.
```

Comment:
I wonder if this should specify this as `nix run . -- [ARGS]` and say that the included HelloWorld application requires on argument.